src/App.js

```diff
@@ -1,25 +1,120 @@
+      <span className="sm:flex-grow"></span>
+      <div className="block md:inline">
+        <a className="text-2xl text-neutral-gray font-bold hover:text-primary" href="/classes">Classes</a>
+        <a className="text-2xl text-neutral-gray font-bold ml-8 hover:text-primary" href="/chats">Chats</a>
+        <a className="text-2xl text-neutral-gray font-bold ml-8 hover:text-primary" href="/profile">Profile</a>
+      </div>
+    </nav>
+  );
+}
+
+function Footer() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="mt-48 px-8 py-5 border-t-2 border-t-gray-200 w-full flex flex-wrap gap-8">
+      <span className="text-neutral-gray text-sm">© 2024 Tomo</span>
+      <a href="/about" className="text-neutral-gray text-sm underline underline-offset-2">About</a>
     </div>
   );
 }
 
-export default App;
+const CLASSES = [
+  {
+    id: 0,
+    title: "COMP_SCI 211-0",
+    description: "Fundamentals of Computer Programming II",
+    members: 67,
+    url: "/classes/comp-sci-211-0"
+  },
+  {
+    id: 1,
+    title: "COMP_ENG 203-0",
+    description: "Introduction to Computer Engineering",
+    members: 48,
+    url: "/classes/comp-eng-203-1"
+  },
+  {
+    id: 2,
+    title: "MATH 228-1",
+    description: "Multivariable Differential Calculus for Engineering",
+    members: 61,
+    url: "/classes/math-228-1"
+  },
+  {
+    id: 3,
+    title: "GEN_ENG 206-1",
+    description: "Honors Engineering Analysis 1",
+    members: 56,
+    url: "/classes/gen-eng-206-1"
+  },
+  {
+    id: 4,
+    title: "CONDUCT 393-0",
+    description: "Orchestral Organizations",
+    members: 77,
+    url: "/classes/conduct-393-0"
+  }
+];
+
+function ClassCard(props) {
+  return (
+    <article className="group relative transition hover:-translate-y-1 h-full cursor-pointer">
+      <a href={props.url}>
+        <div className="border-2 border-gray-200 rounded-2xl p-8 h-full group-hover:shadow-md">
+          <h2 className="text-3xl text-neutral-black font-bold font-display">{props.title}</h2>
+          <p className="text-md text-neutral-gray">{props.description}</p>
+          <p className="mt-1 text-md text-neutral-gray">{props.members} members</p>
+        </div>
+      </a>
+      <button className="text-neutral-black hover:text-red-500 absolute top-4 right-4"
+              aria-label={`Remove ${props.title}`}
+              onClick={props.handleRemove}>
+        <FontAwesomeIcon icon={faTrashCan} size="lg"/>
+      </button>
+    </article>
+  );
+}
+
+function App() {
+  const [classes, setClasses] = useState(CLASSES);
+
+  const handleRemove = (index) => {
```

Comment:
this is nicely done! one thing i'd suggest for further extensibility: right now, you're using index to remove specific classes from the list, but index might change if the order of classes in the array changes. i'd recommend using c.id to filter classes instead, something like setClasses(classes.filter(c => c.id !== id)).